api-report/odsp-driver.api.md

```diff
@@ -141,7 +142,7 @@ export class OdspDocumentServiceFactoryCore implements IDocumentServiceFactory {
     //
     // (undocumented)
     protected createDocumentServiceCore(resolvedUrl: IResolvedUrl, odspLogger: TelemetryLogger, cacheAndTrackerArg?: ICacheAndTracker, clientIsSummarizer?: boolean): Promise<IDocumentService>;
-    getRelayServiceSessionInfo(resolvedUrl: IResolvedUrl): Promise<ISocketStorageDiscovery | undefined>;
+    getRelayServiceSessionInfo(resolvedUrl: IResolvedUrl): Promise<ISocketStorageDiscoveryParts | undefined>;
```

Comment:
Are we tied to this name? We don't use the term RelayService elsewhere AFAIK.

Reply:
We generally refer to push session as relayServiceSession. We use this term in comments in odsp driver. It is better than just using the word "session" because session could be of anything.